docs/hosting/server-setups/digital-ocean.md

```diff
@@ -88,6 +102,8 @@ GENERIC_TIMEZONE=Europe/Berlin
 SSL_EMAIL=example@example.com
 ```
 
+For more information on n8n environment variables, refer to [Environment variables](/hosting/environment-variables/).
```

Comment:
```suggestion
For more information on n8n environment variables, read the [Environment variables](/hosting/environment-variables/) documentation.
```

This is persona, just how I have always constructed reference links, but if you have your own style, go for it.